exec/src/extends.mli

```diff
@@ -0,0 +1,21 @@
+(* FreeSpec
+ * Copyright (C) 2018–2019 ANSSI
+ *
+ * Contributors:
+ * 2019 Thomas Letan <thomas.letan@ssi.gouv.fr>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *)
+
+val register_interface: string list -> (string * Interfaces.effectful_semantic) list -> unit
```

Comment:
Needs for a documentation.

Reply:
A shorter version of [this](https://github.com/ANSSI-FR/FreeSpec/wiki/Introduction-to-FreeSpec.Exec) could make a good documentation for the `Extends` mli file, don’t you think?